lua/bufferline/render.lua

```diff
@@ -138,7 +138,7 @@ local function render()
         icon = number_text .. (#number_text > 1 and '' or ' ')
       else
         local iconChar, iconHl = get_icon(buffer_name, vim.fn.getbufvar(buffer_number, '&filetype'))
-        iconPrefix = status == 'Inactive' and hl('BufferInactive') or hl(iconHl or ('Buffer' .. status))
+        iconPrefix = status ~= 'Current' and hl('BufferInactive') or hl(iconHl or ('Buffer' .. status))
```

Comment:
Sometimes a status could be `Visible` but still technically not `Current`, and so the icon would be displayed like the `Current` buffer, but the rest of the buffer would be displayed like `Visible`.